cmd/generate.go

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/srl-wim/container-lab/clab"
+	"gopkg.in/yaml.v2"
+)
+
+var interfaceFormat = map[string]string{
+	"srl":    "e1-%d",
+	"ceos":   "eth%d",
+	"linux":  "eth$d",
+	"bridge": "veth%d",
+}
+var supportedKinds = []string{"srl", "ceos", "linux", "bridge", "sonic", "crpd"}
+
+const (
+	defaultSRLType     = "ixr6"
+	defaultNodePrefix  = "node"
+	defaultGroupPrefix = "group"
```

Comment:
since by default we build a CLOS topo, maybe we should use the group prefix value as `tier`?
That will make sense to a user, since CLOS topos break down by tiers, which will make a structure:

* tier1
* tier2
* etc

Reply:
I have no opinion on the default value, it can be changed with `--group-prefix` anyways